taui/src/components/main-page.js

```diff
@@ -108,6 +108,7 @@ export default class MainPage extends React.PureComponent<Props> {
   render () {
     const p = this.props
     const mapScreenClass = p.isLoading ? 'map-screen isLoading' : 'map-screen'
+    const neighborhoods = p.data.neighborhoods ? p.data.neighborhoods.features : null
```

Comment:
It would probably be better to compute the neighborhood count in a reducer and passing that, rather than passing all the neighborhoods to the `Dock` component, since it doesn't need them all otherwise, just the current page. I can do that in another PR.

Reply:
That makes sense. Thank you for taking care of!